src/scss/components/utility/_utility.scss

```diff
@@ -30,9 +37,42 @@
         &:hover,
         &:focus {
           color: $vpue-color-off-white;
+          background-color: $su-color-bright-red;
```

Comment:
So only on the homepage? is the underline enough?

Reply:
@buttonwillowsix we need a hover state for that button. Should I leave in the black?